packages/bierzo-wallet/src/routes/payment/components/ReceiverAddress/index.tsx

```diff
@@ -4,52 +4,46 @@ import { Theme } from '@material-ui/core';
 import Paper from '@material-ui/core/Paper';
 import TextField from '@material-ui/core/TextField';
 import { makeStyles } from '@material-ui/styles';
+import Block from 'medulas-react-components/lib/components/Block';
+import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
 
 interface Props {
   readonly positionedClass?: string;
 }
 
 const useStyles = makeStyles((theme: Theme) => ({
-  paperRoot: {
+  paper: {
     display: 'flex',
     flexDirection: 'column',
     width: '100%',
-    padding: '4rem',
+    padding: '40px',
```

Comment:
Again, this should be solved with `Block` component which is based on:
https://next.material-ui.com/system/spacing/

Reply:
I'm not sure if I should delete the Paper component completely and apply its styles to Block, or surround Paper with Block. What do you think?

Reply:
I would say to apply styles directly to Block because you are simplifying things and also you avoid the usage of an extra component.